docs/ref/contrib/postgres/operations.txt

```diff
@@ -59,3 +59,35 @@ the ``django.contrib.postgres.operations`` module.
 .. class:: UnaccentExtension()
 
     Installs the ``unaccent`` extension.
+
+``Create extension using migrations``
+=====================================
+
+You can create a PostgreSQL Extension in your database using a migration file.
+The following example will cover creating of hstore extension,
+but same usage principles apply for all other extensions, mentioned above.
+
+1. Add ``'django.contrib.postgres'`` in your :setting:`INSTALLED_APPS`.
+2. Setup the hstore extension in PostgreSQL before the first ``CreateModel``
+   or ``AddField`` operation by adding a migration with the
+   :class:`~django.contrib.postgres.operations.HStoreExtension` operation.
+   For example::
+
+        from django.contrib.postgres.operations import HStoreExtension
+
+        class Migration(migrations.Migration):
+            ...
+
+            operations = [
+                HStoreExtension(),
+                ...
```

Comment:
These dots suggests there is more you should do. Is that mandatory? Wouldn't your example (without the dots) be good enough as a single migration?

(Note: I'm not an expert on postgres extensions, so I might be wrong).